region/lowernorfair/east/Red Kihunter Shaft.json

```diff
@@ -421,6 +434,16 @@
         {"refill": ["Energy", "Missile", "Super"]}
       ]
     },
+    {
+      "link": [3, 3],
+      "name": "Crystal Flash",
+      "requires": [
+        {"heatFrames": 20},
+        "h_canHeatedCrystalFlash",
+        {"heatFrames": 20}
+      ],
+      "note": "Perform the Crystal Flash to the left, away from the door, to prevent the Kihunter from coming down."
```

Comment:
Probably fine.  The manip strats would want to reload the room before going to 5.